bindings/python/unicorn/x86_const.py

```diff
@@ -248,7 +248,9 @@
 UC_X86_REG_GDTR = 243
 UC_X86_REG_LDTR = 244
 UC_X86_REG_TR = 245
-UC_X86_REG_ENDING = 246
+UC_X86_REG_FPTAGS = 246
+UC_X86_REG_FPCW = 247
```

Comment:
the order is incorrect here: FPCW=246, FPTAGS=247. if you should already detected this if you had a testcase for these new registers.